src/modules/fancyzones/editor/FancyZonesEditor/Properties/Resources.resx

```diff
@@ -353,15 +353,23 @@
     <comment>Title for concept behind Merging two zones together or removing an zone</comment>
   </data>
   <data name="KeyboardControlsName" xml:space="preserve">
-    <value>Keyboard Navigation:</value>
+    <value>Keyboard navigation:</value>
   </data>
   <data name="KeyboardControlsDescription" xml:space="preserve">
     <value>

```

Comment:
This newline is also passed to the UI. Is that wanted?

Reply:
which newline ?

Reply:
Oh, got it. checking how it looks like without it

Reply:
yes, this is needed